libraries/iio/Makefile

```diff
@@ -0,0 +1,52 @@
+ifeq ($(OS), Windows_NT)
+remove_fun = del /S /Q $(subst /,\,$1)
+else
+remove_fun = rm -rf $(1)
+endif
+
+# Local variables
+TINYIIOD_VERSION_MAJOR = 0
+TINYIIOD_VERSION_MINOR = 1
+TINYIIOD_VERSION = $(TINYIIOD_VERSION_MAJOR).$(TINYIIOD_VERSION_MINOR)
+
+BUFF_SIZE ?= 0x1000
+STD_TYPES ?= true
+
+# Build settings
+IIO_DEFS = -D TINYIIOD_VERSION_MAJOR=$(TINYIIOD_VERSION_MAJOR)		\
+	   -D TINYIIOD_VERSION_MINOR=$(TINYIIOD_VERSION_MINOR)		\
+	   -D TINYIIOD_VERSION_GIT=0x$(shell git rev-parse --short HEAD)\
+	   -D IIOD_BUFFER_SIZE=$(BUFF_SIZE)
+
+ifeq (true,$(strip $(STD_TYPES)))
+IIO_DEFS += -D_USE_STD_INT_TYPES
+endif
+
+IIO_DIR		= $(realpath .)
```

Comment:
I'm not very familiar with makefile syntax but i was wondering if `.` could point to different directory than `libraries/iio`. I tend to think that it's not possible so this should be ok.

Reply:
So, when "make -C libraries/iio" is run from no-os makefile directory will change to libraries/iio -> run make -> go back to last directory.
So yes, "." will point to libraries/iio